db/seeds.rb

```diff
@@ -93,14 +113,21 @@
                   tenant_id: bridge_builders1.id
                   )
 
+timmys_vaccines_nigeria.photos << Photo.create!(
+        image: File.new( "#{Rails.root}/app/assets/images/timmys_vaccines.jpg")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Space inside parentheses detected.